draft-terbu-oauth-sd-jwt-vc.md

```diff
@@ -197,18 +197,20 @@ SD-JWT VCs.
 
 #### `type` claim {#type-claim}
 
-This specification defines the JWT claim `type`. The `type` claim is used
-to express the type of the JSON object that is secured by the
-JWT. The `type` value MUST be a case-sensitive `StringOrURI` value.
+This specification defines the JWT claim `type`. The `type` claim value is the
+identifier for a pre-defined bundle of claims associated with certain
+policies for enrollment and issuance that the respective SD-JWT VC
+conforms to. The `type` value MUST be a case-sensitive `StringOrURI` value.
```

Comment:
```suggestion
This specification defines the JWT claim `type`. The `type` claim value is the
identifier for a pre-defined bundle of claims that the respective SD-JWT VC
conforms to. The `type` value MUST be a case-sensitive `StringOrURI` value.
```

I don't really understand what `associated with certain policies for enrollment and issuance` means..?

Reply:
I agree, I don't fully understand either I think I have a clue but imo, it sounds a bit too difficult to parse for most readers.

Reply:
I'd prefer my suggestion below.